src/test/java/mate/academy/dao/UserDaoTest.java

```diff
@@ -0,0 +1,81 @@
+package mate.academy.dao;
+
+import java.util.Optional;
+import java.util.Set;
+import mate.academy.dao.impl.RoleDaoImpl;
+import mate.academy.dao.impl.UserDaoImpl;
+import mate.academy.model.Role;
+import mate.academy.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class UserDaoTest extends AbstractTest {
+    private static final Role.RoleName ROLE = Role.RoleName.USER;
+    private static final String EMAIL = "someemail@gmail.com";
+    private static final String PASSWORD = "password";
+    private UserDao userDao;
+    private RoleDao roleDao;
+    private User user;
+    private Role role;
+
+    @BeforeEach
+    void setUp() {
+        roleDao = new RoleDaoImpl(getSessionFactory());
+        role = new Role();
+        role.setRoleName(ROLE);
+        roleDao.save(role);
```

Comment:
You have 5 test cases (methods) in this class. So this line will be executed 5 times. Do you need to insert new role into DB before each test case?